ietf/templates/nomcom/view_feedback.html

```diff
@@ -54,6 +54,14 @@ <h3 class="mt-5" id="declined">Declined each nominated position</h3>
                     </tr>
                 {% endfor %}
             </tbody>
+	    <tfoot>
+		<td></td>
+		<td></td>
+		<th>Totals</h></th>
```

Comment:
th needs scope

Reply:
Do you think `col` is right?  I didn't  so I left it out